test/conftest.py

```diff
@@ -164,6 +164,17 @@ def orcid_tokens() -> SQLAlchemyOrcidTokens:
     return SQLAlchemyOrcidTokens(db)
 
 
+@fixture
+def registered_handler_names():
```

Comment:
Can you extract this change into a new PR? It's unrelated to this change and keeps happening on other builds. We should get it in first.

Reply:
Unless we're about to merge!

Reply:
(Do separate PRs in the future though, keeps unrelated changes separate and is clearer.)